Lib/logging/config.py

```diff
@@ -463,7 +463,7 @@ def configure_custom(self, config):
             c = self.resolve(c)
         props = config.pop('.', None)
         # Check for valid identifiers
-        kwargs = dict([(k, config[k]) for k in config if valid_ident(k)])
+        kwargs = {k: config[k] for k in config if valid_ident(k)}
```

Comment:
This should use the iterable-of-pairs form instead of creating an intermediate dictionary:
dict((k, config[k]) for k in config if valid_ident(k))